daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/transaction/PartialTransaction.scala

```diff
@@ -233,11 +241,11 @@ private[lf] case class PartialTransaction(
           nodes
         )
         .fold(
-          s => Left(s"Cannot serialized the transaction: $s"),
-          tx => Right(Right(SubmittedTransaction(tx)))
+          s => SerializationError(s"Cannot serialized the transaction: $s"),
```

Comment:
I suggest adding flavor text at the last possible moment.

```suggestion
          message => SerializationError(message),
```

(BTW, you also meant "serialize", not "serialized".)